50-fixed-percentage-off-scale

burtscriptor opened the pull request:
Changed line 231 from 

document.getElementById('outOfRange').textContent = `>${selectedUnit.max + ' ' + selectedUnit.label}: ${aboveMax}%`

to

document.getElementById('outOfRange').textContent = `>${selectedUnit.max + ' ' + selectedUnit.label}: ${(aboveMax / (recordPack.data.length) * 100).toFixed(0)}%`

to calculate the percentage of measurements above the the max speed for each unit.

NickRoach reviewed (approved):
LGTM 👍

NickRoach commented:
My man! Nice work. Can you please rebase your branch to develop though? It looks like it's based on main. I should have explained that the way this project is set up is that we merge into develop first, and that branch gets deployed to testing environment, then when we're happy we merge develop --> master

burtscriptor commented:
Yeh sorry about that, new guys what do they know. Ok so i've rebased to develop ...
 
blackheath-wind-meter % git rebase develop
Successfully rebased and updated refs/heads/50-fix-off-scale-percentage.
~macbook-air/blackheath-wind-meter % git branch
* 50-fix-off-scale-percentage
  develop
  main

..and now I think i need to do the pull request?

NickRoach commented:
Nice :) No need to make a new PR, you can just force push and re-use this one

NickRoach commented:
And it's ok, not a new guy thing. You couldn't have known

NickRoach reviewed (approved):
LGTM 👍
